packages/eslint-plugin/docs/rules/no-implicit-any-catch.md

```diff
@@ -0,0 +1,57 @@
+# Disallow usage of the implicit `any` type in catch clauses (`no-implicit-any-catch`)
+
+Using the `any` type defeats the purpose of using TypeScript.
+When `any` is used, all compiler type checks around that value are ignored.
+
+The noImplicitAny flag in TypeScript does not cover this due to backwards compatibility reasons.
+
+## Rule Details
+
+This rule requires an explicit type to be declared in the catch clause error argument.
+
+The following pattern is considered a warning:
+
+```ts
+try {
+  // ...
+} catch (e) {
+  // ...
+}
+```
+
+The following patterns are not warnings:
+
+```ts
+try {
+  // ...
+} catch (e) {
+  // ...
+}
+```
+
+```ts
+try {
+  // ...
+} catch (e) {
```

Comment:
```suggestion
} catch (e: unknown) {
```

Reply:
this is sadly impossible to commit as long as prettier styling is enforced (which automatically strips these types)

Reply:
/* : unknown */ until then?

Reply:
good idea.